docs/about.md

```diff
@@ -11,22 +11,36 @@
 The **project team**:
 
 * [**Aaron Chong Jun Hao**](https://github.com/acjh): _Project Mentor_ since Jul 2018, _Team Lead_ for Aug 2017 - Jun 2018
+* [**Ang Ze Yu**](https://www.github.com/ang-zeyu): _Project Member_ since Jan 2020
 * [**Chng Zhi Xuan**](https://github.com/Chng-Zhi-Xuan): _Project Member_ since May 2018
 * [**Chua Yun Zhi Nicholas**](https://github.com/nicholaschuayunzhi): _Project Member_ since Jan 2018
 * [**Damith C. Rajapakse**](https://www.comp.nus.edu.sg/~damithch/): _Project Mentor_ since Aug 2016
+* [**Daryl Tan**](https://www.github.com/openorclose): _Project Member_ since May 2019
 * [**Jamos Tay**](https://github.com/jamos-tay): _Project Member_ since Aug 2018
-* [**Lin Si Jie**](https://www.github.com/sijie123): _Project Member_ since Jan 2019
-* [**Lu Yang Kenneth**](https://github.com/luyangkenneth): _Project Member_ since Jan 2019
-* [**Maddi Aadyaa**](https://www.github.com/amad-person): _Project Member_ since Jan 2019
-* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Member_ since Jan 2019
+* [**Labayna Neil Brian Narido**](https://www.github.com/nbriannl): _Project Member_ since Jan 2020
+* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Mentor_ since Jan 2020, _Project Member_ since Jan 2019 - Aug 2019
```

Comment:
we can use `for` here too - 
```suggestion
* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Mentor_ since Jan 2020, _Project Member_ for Jan 2019 - Aug 2019
```